docs/source/software/conda.rst

```diff
@@ -7,6 +7,9 @@ Working with Conda/Miniconda/Anaconda
 and a few other useful packages. `Anaconda <https://docs.anaconda.com/anacondaorg/faq/>`_ is also a package manager that has a much larger number of packages that you can install.
 A question that frequently comes up is "Should I use Anaconda or Miniconda?" The Conda documentation site has a topic that can help you to decide which package manager to use: https://docs.conda.io/projects/conda/en/latest/user-guide/install/download.html#anaconda-or-miniconda.
 
+.. note::
+  It is not recommended to build your Miniconda and virtual Conda environments inside your /home directory due to its limited space qouta (see :ref:`discovery_storage`). Use the /work or /scratch file system instead. If your group needs access to /work, the group PI can request it using: `New Storage Space request <https://bit.ly/NURC-NewStorage>`_ .
```

Comment:
Miniconda and virtual Conda environments - > Miniconda and Conda virtual environments

Reply:
I am not sure if /scratch should be mentioned because it is temporary storage and sometimes conda environments need to persist for. the duration of the project.